packages/jellyfish-transaction/src/script/defi/dftx_token.ts

```diff
@@ -8,8 +8,12 @@ import { BufferComposer, ComposableBuffer } from '../../buffer/buffer_composer'
  * TokenMint DeFi Transaction
  */
 export interface TokenMint {
-  tokenId: number // -------------------| VarUInt{1-9 bytes}
-  amount: BigNumber // -----------------| 8 bytes
+  balances: TokenBalance[] // ----------| c = VarUInt{1-9 bytes}, + c x TokenBalance
+}
+
+export interface TokenBalance {
+  token: number // ---------------------| 4 bytes unsigned
+  amount: BigNumber // -----------------| 8 bytes unsigned
 }
```

Comment:
you should use the existing https://github.com/DeFiCh/jellyfish/blob/3fb5abc3f8db0e272e9940d22224ff747d0bd728/packages/jellyfish-transaction/src/script/defi/dftx_balance.ts#L9-L12

from `dftx_balanace`